projects/demo/src/modules/components/combo-box/examples/1/index.ts

```diff
@@ -44,16 +45,20 @@ export class TuiComboBoxExample1 {
 
     readonly testValue = new FormControl(databaseMockData[1]);
 
-    onSearchChange(searchQuery: string) {
+    onSearchChange(searchQuery: string | null) {
         this.search$.next(searchQuery);
     }
 
+    extractValueFromEvent(event: Event): string {
+        return (event.target as EventTarget & {value?: string})?.value ?? '';
```

Comment:
Typecast it as `HTMLInputElement` if you're casting anyway.

Reply:
fixed